lighthouse-core/runner.js

```diff
@@ -325,7 +325,12 @@ class Runner {
         ...sharedAuditContext,
       };
 
-      const product = await audit.audit(artifacts, auditContext);
+      const requiredArtifacts = audit.meta.requiredArtifacts
```

Comment:
can you add a comment about narrowing down the artifacts to just what is requested by the audit

Reply:
👍 done